### ISSUE TITLE
Missing i18n

### ISSUE THREAD

**Issue by DirkHoffmann** [indico/indico#5541]
The texts of `_DEFAULT_RESTRICTED_DISCLAIMER` and `_DEFAULT_PROTECTED_DISCLAIMER` should be i15d. As their display is not direct, but goes to `SettingsProxy`, I am not sure how to do it properly. Probably it should be a `_()` around the text used by `SettingsProxy`, but the `_DEFAULT…` constants also need to be referenced to transifex.

https://github.com/indico/indico/blob/5c9fb38207175932a3c285796ca3b82ef0ff56bf/indico/modules/legal/__init__.py#L18-L23

**Comment by ThiefMaster** [comment]
Nope, there's no easy/good solution for adding i18n there (like for category titles, event titles, regform field labels, etc.). These "disclaimers" are admin-editable text after all.

I think the best option for now is to go to the admin area and include all relevant languages in the same field...

**Comment by DirkHoffmann** [comment]
I see. But right now it is mixed in the popup balloon (first paragraph i15d, next in english (or customized)), which looks weird.
Remove it from the first paragraph then? :thinking:
